openmetadata-ui/src/main/resources/ui/src/utils/ToastUtils.ts

```diff
@@ -81,9 +78,7 @@ export const showErrorToast = (
  * @param autoCloseTimer Set the delay in ms to close the toast automatically. `Default: 5000`
  */
 export const showSuccessToast = (message: string, autoCloseTimer = 5000) => {
-  toast.success(message, {
-    autoClose: autoCloseTimer,
-  });
+  useAlertStore.getState().addAlert({ type: 'success', message });
```

Comment:
We have removed autoCloseTimer option can we have that in new component

Reply:
Yes we are considering this and we will take care of this.